src/main.cpp

```diff
@@ -6110,6 +6110,28 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         }
     }
 
+    else if (strCommand == NetMsgType::NOTFOUND) {
+        vector<CInv> vInv;
+        vRecv >> vInv;
+        if (vInv.size() > MAX_INV_SZ)
+        {
+            LOCK(cs_main);
+            Misbehaving(pfrom->GetId(), 20);
+            return error("message notfound size() = %u", vInv.size());
+        }
+
+        for (unsigned int nInv = 0; nInv < vInv.size(); nInv++)
+        {
+            const CInv &inv = vInv[nInv];
+
+            boost::this_thread::interruption_point();
+            if (inv.type == MSG_TX || inv.type == MSG_WITNESS_TX)
+                 LogPrint("tx", "notfound: %s from peer=%d\n", inv.ToString(), pfrom->id);
```

Comment:
Do you really want to log a line for every single record? This can be up to 50,000 in one message.